examples/text-sentiment/models/text_sentiment/config.yml

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-block_id: 8f72161-c0e4-49b0-8fd0-7587b3017a35
```

Comment:
We should probably keep at least one of these for testing (maybe you have below)

Reply:
I noted in the PR description that the zip files still test backwards compatibility since they still use `block_id`. There are tests to load those. Hoping that's sufficient.